src/course-about/course-overview/CourseOverview.test.tsx

```diff
@@ -0,0 +1,116 @@
+import { getAuthenticatedUser } from '@edx/frontend-platform/auth';
+import { getConfig } from '@edx/frontend-platform';
+
+import { render, screen } from '@src/setupTest';
+import messages from '../messages';
+import { CourseOverview } from '.';
+
+jest.mock('@edx/frontend-platform/auth', () => ({
+  getAuthenticatedUser: jest.fn(),
+}));
+
+jest.mock('@edx/frontend-platform', () => ({
+  getAuthenticatedUser: jest.fn(() => ({ username: 'test-user', roles: [] })),
+  getConfig: jest.fn(),
+}));
+
+const mockGetAuthenticatedUser = getAuthenticatedUser as jest.Mock;
+const mockGetConfig = getConfig as jest.Mock;
+
+const mockCourseId = 'course-v1:TestX+Test101+2023';
+
+describe('CourseOverview', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+    mockGetConfig.mockReturnValue({
+      LMS_BASE_URL: process.env.LMS_BASE_URL,
+      STUDIO_BASE_URL: process.env.STUDIO_BASE_URL,
+    });
+    mockGetAuthenticatedUser.mockReturnValue(null);
+  });
+
+  describe('Content rendering', () => {
+    it('renders overview content when provided', () => {
+      const overviewData = '<p>Course overview content</p>';
+      render(<CourseOverview overviewData={overviewData} courseId={mockCourseId} />);
+
+      expect(screen.getByText(overviewData.replace(/<[^>]*>?/g, ''))).toBeInTheDocument();
+    });
```

Comment:
Having the regex in here makes this test feel a bit unclear to me. I understand that it's just because `getByText` doesn't see the `p` tags, so we don't want to test the HTML part but we still want to put HTML in as the prop because we use it when we `dangerouslySetInnerHTML`.

How do you feel about something like


```suggestion
    it('renders overview content when provided', () => {
      const overviewText = 'Course overview content';
      const overviewData = `<p>${overviewText}</p>`;
      render(<CourseOverview overviewData={overviewData} courseId={mockCourseId} />);

      expect(screen.getByText(overviewText)).toBeInTheDocument();
    });
```